profile-widget/profilewidget2.cpp

```diff
@@ -255,6 +255,17 @@ void ProfileWidget2::divesChanged(const QVector<dive *> &dives, DiveField field)
 
 void ProfileWidget2::actionRequestedReplot(bool)
 {
+	/* this is called vai infoboxChanged, therefore in currentState==PROFILE
```

Comment:
```suggestion
	/* this is called via infoboxChanged, therefore in currentState==PROFILE
```

Also, the 'hk' doesn't seem to be super useful